tfhe/src/integer/ciphertext/mod.rs

```diff
@@ -1,7 +1,10 @@
 //! This module implements the ciphertext structures.
+use crate::shortint::ciphertext::{BootstrapKeyswitch, KeyswitchBootstrap};
 use crate::shortint::{
-    CiphertextBig as ShortintCiphertext, CompressedCiphertextBig as CompressedShortintCiphertext,
+    CiphertextBase, CiphertextBig, CiphertextSmall, CompressedCiphertextBig,
+    CompressedCiphertextSmall, PBSOrderMarker,
 };
+type ShortintCiphertext = CiphertextBig;
```

Comment:
is this type alias still required ?

Reply:
it was but  only used once so I "inlined" it